src/main/java/org/jenkinsci/test/acceptance/junit/WithPlugins.java

```diff
@@ -80,6 +82,31 @@
         @Inject(optional=true) @Named("neverReplaceExistingPlugins")
         boolean neverReplaceExistingPlugins;
 
+        /*package for testing*/ static List<PluginSpec> combinePlugins(List<WithPlugins> wp) {
```

Comment:
There is a `@VisibleForTesting` annotation somewhere, not sure if accessible in this project.